lib/hider/accounts.ex

```diff
@@ -58,11 +74,43 @@ defmodule Hider.Accounts do
 
   """
   def create_user(attrs \\ %{}) do
+    Multi.new()
+    |> Multi.put(:attrs, attrs)
+    |> Multi.insert(:user, &do_create_user_changeset/1)
+    |> Multi.insert_all(:trigrams, Trigram, &do_create_trigrams/1)
```

Comment:
Create the trigrams to allow fuzzy search (this takes a long long time, best if run as another job, there is not much way around it, keeping this `safer` and `fast`, fast hashes are easily crackable now a days)